test/cypress/integration/widgets/deployment_button_spec.ts

```diff
@@ -161,6 +163,23 @@ describe('Create Deployment Button widget', () => {
         verifyDeploymentInstallStarted(deploymentId);
     });
 
+    it('blueprint with custom install workflow', () => {
```

Comment:
1. I'm wondering why you decided to create another test case instead of extending 'allows to deploy and install a blueprint'? I'm not saying it's bad or good.

2. If you are willing to keep this test case, then to be inline with the naming convention I think the better would be to name it:
```suggestion
    it('allows to set install workflow parameters', () => {
```

3. Setting parameters is one thing, but maybe worth to check if they are passed down to the manager also? You could also change some of the toggles.

Reply:
when I combine a `simple` blueprint:
```
tosca_definitions_version: cloudify_dsl_1_3

description: >
  This is very simple blueprint

imports:
  - http://www.getcloudify.org/spec/cloudify/5.0.5/types.yaml

inputs:
  server_ip:
    description: >
      The ip of the server the application will be deployed on.
    display_label: Server IP
  hidden:
    hidden: true
    default: 0

node_templates:
  vm:
    type: cloudify.nodes.Compute
    properties:
      ip: { get_input: server_ip }
```
with `custom install workflow parameters` blueprint:
```
tosca_definitions_version: cloudify_dsl_1_3

plugins:
    default_workflows:
        executor: central_deployment_agent
        install: false

workflows:
    install:
        mapping: default_workflows.cloudify.plugins.workflows.install
        is_cascading: false
        parameters:
            xxx:
                default: blabla
            yyy:
                default: blabla
```
I'm getting this blueprint:
```
tosca_definitions_version: cloudify_dsl_1_3

description: >
  This is very simple blueprint

imports:
  - http://www.getcloudify.org/spec/cloudify/5.0.5/types.yaml

inputs:
  server_ip:
    description: >
      The ip of the server the application will be deployed on.
    display_label: Server IP
  hidden:
    hidden: true
    default: 0

node_templates:
  vm:
    type: cloudify.nodes.Compute
    properties:
      ip: { get_input: server_ip }

plugins:
  default_workflows:
    executor: central_deployment_agent
    install: false

workflows:
  install:
    mapping: default_workflows.cloudify.plugins.workflows.install
    is_cascading: false
    parameters:
      xxx:
        default: blabla
      yyy:
        default: blabla
```
But I do not get the workflow parameters from the concatenated blueprint.
I'm afraid to waste too much time on modifying the blueprint. So my solution will be to use the current blueprint I have. I will combine the tests.

Reply:
I address points 1 and 2 (I combined the tests): [ad275d1](https://github.com/cloudify-cosmo/cloudify-stage/pull/1812/commits/ad275d1784457a61f414f5ace897fc0f7b4b115f)

I'm working now on point 3: to verify changing and saving `custom install workflow parameters` and `dry run`

Reply:
I addressed point 3:
- [48d8a2d](https://github.com/cloudify-cosmo/cloudify-stage/pull/1812/commits/48d8a2d816fd6ff323940c556607703c2183c5fd)
- [8b728c4](https://github.com/cloudify-cosmo/cloudify-stage/pull/1812/commits/8b728c4119f8a5d530235a4f3a51049ee2e3d889)